src/PolyhedralGeometry/markov_basis.jl

```diff
@@ -0,0 +1,68 @@
+@doc Markdown.doc"""
+    markov_basis(S; use_kernel=true)
+    markov_basis(P::Polyhedron{fmpq})
+
+
+Return a Markov basis for the lattice $L$. Where $L$ is
+- either the integer kernel of a matrix when either `use_kernel=true` or the
+  argument is of type `Polyhedron{fmpq}`,
+- otherwise $L$ is given by the span of the collection of integer vectors `S`.
```

Comment:
is `use_kernel` allowed then, too? right now the docstring is written in a way that suggests it is